test/helpers/utils.js

```diff
@@ -105,6 +105,20 @@ const getIteration = async (voteManager, stakeManager, staker) => {
   return 0;
 };
 
+const getFalseIteration = async (voteManager, stakeManager, staker) => {
+  const numStakers = await stakeManager.getNumStakers();
+  const stakerId = staker.id;
+  const influence = await stakeManager.getInfluence(stakerId);
+
+  const { biggestInfluence } = await getBiggestInfluenceAndId(stakeManager);
+  const randaoHash = await voteManager.getRandaoHash();
+  for (let i = 0; i < 10000000000; i++) {
+    const isElected = await isElectedProposer(i, biggestInfluence, influence, stakerId, numStakers, randaoHash);
+    if (isElected && i > 1) return (i - 1);
```

Comment:
what if iselected= TRUE FOR i = 1
and iselected=true for i=2

Reply:
I think, 
for iselected = true and i = 2, it should return 1
for iselected = true and i = 1, it should return 0
for iselected = true and i = 0, it should return 1 since negative numbers are not accepted

Reply:
for iselected = true and i = 2, it should return 1

thats the problem

it will return 1. but 1 can be elected.

Reply:
Oh okay